src/app/app.component.ts

```diff
@@ -38,10 +39,15 @@ export class AppComponent {
     return this.authService.claims?.email;
   }
 
-  constructor(private authService: AuthService, private dataService: DataService, private trainrunService: TrainrunService, private trainrunSectionService: TrainrunSectionService) {
+  constructor(private authService: AuthService,
+              private uiInteractionService: UiInteractionService,
+              private dataService: DataService,
+              private trainrunService: TrainrunService,
+              private trainrunSectionService: TrainrunSectionService) {
     if (!this.disableBackend) {
       this.authenticated = authService.initialized;
     }
+    this.uiInteractionService.updateLightDark();
```

Comment:
Can we move this logic at the same place where we load active theme ? I don't find any easy solution. Otherwise can be ignored

Reply:
Removed it - no longer required :-)

Reply:
Please retest :-) Thanks